snowplow-tracker/src/main/java/com/snowplowanalytics/core/tracker/InstallReferrerDetails.kt

```diff
@@ -72,7 +71,7 @@ class InstallReferrerDetails(
                                     googlePlayInstantParam = response.googlePlayInstantParam
                                 )
                                 callback(referrer)
-                            } catch (_: RemoteException) {
+                            } catch (_: Throwable) {
```

Comment:
This is fine, but I wonder if we should catch the `NoSuchMethodError` exception specifically to give a more concrete error message saying that the problem likely is that they need to update the InstallReferrer to v1.1?